Gemfile

```diff
@@ -2,3 +2,12 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in amazon_ssa_support.gemspec
 gemspec
+
+gem "aws-sdk"
```

Comment:
Extra lines are for 'bundle install' purpose, will remove them in the next commit.